src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/SlhDsa/SlhDsaContractTests.cs

```diff
@@ -0,0 +1,221 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Formats.Asn1;
+using System.Linq;
+using System.Security.Cryptography.Asn1;
+using Xunit;
+
+using static System.Security.Cryptography.SLHDsa.Tests.SlhDsaTestHelpers;
```

Comment:
`using static` is one of the many "this saves a fraction of a second when writing code, this has very little cost when using an IDE, this adds an astronomical cost when an IDE isn't available" features.

I think we're better off not using it.  Please remove it from everywhere in this PR, and just pay the extra 18 characters to call it using normal static method invocation syntax.